client/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import MuiCard from '@mui/material/Card';
+import Typography from '@mui/material/Typography';
+import { CardActionArea } from '@mui/material';
+import theme from '../../theme';
+
+export default function Card(props: any) {
```

Comment:
we shouldn't have `any`s if possible
try something like one of the last 2 options [here](https://blog.logrocket.com/using-react-children-prop-typescript/)